C++/test/algorithm/dynamic_programming/coin_change.cpp

```diff
@@ -0,0 +1,15 @@
+#define CATCH_CONFIG_MAIN
+#include "third_party/catch.hpp"
+#include "algorithm/dynamic_programming/coin_change.hpp"
+
+using std::vector;
+
+TEST_CASE("Base Cases", "[coin_change]") {

```

Comment:
You should also use base case to test for cases with no values or a single value.